apps/worker/services/cleanup/pulls.py

```diff
@@ -112,12 +98,12 @@ def cleanup_flare(context: CleanupContext, batch_size: int = DELETE_FILES_BATCHS
                     _flare_storage_path=None
                 )
 
-            total_files_processed += len(batch_of_id_path_pairs)
-            context.add_progress(cleaned_files=len(successful_deletions), model=Pull)
+            cleaned_file_flares += len(successful_deletions)
+            context.add_progress(cleaned_files=cleaned_file_flares, model=Pull)
```

Comment:
```suggestion
            context.add_progress(cleaned_files=len(successful_deletions), model=Pull)
```

Otherwise it would over-count. Was this bug also present in my other PR?